addons/mail/static/src/core/common/composer.xml

```diff
@@ -216,12 +216,12 @@
 
     <t t-name="mail.Composer.suggestionThread">
         <strong class="px-2 py-1 align-self-center flex-shrink-0 text-truncate">
-            <i t-attf-class="fa #{option.thread.channel?.parent_channel_id ? 'fa-comments-o' : 'fa-hashtag'} me-2"/>
-            <t t-if="option.thread.channel?.parent_channel_id">
```

Comment:
why did we remove the guards on channel? What happens if the channel get deleted from the store?